<?xml version="1.0"?>
<launch>
	<arg name="pkg_path" default="beginner_tutorials"/>
	<node pkg="$(arg pkg_path)" type="state_listener.py" name="button_listener" output="screen">
		<param name="sub_topic_name" type="string" value="iqr/callbutton/state" />
                <param name="sub_topic_name2" type="string" value="iqr/callbutton/goal" />
		<param name="time_out" type="double" value="10" />
		<param name="looprate" type="int" value="1" />
                <param name="mqtt_listener/client_id" type="string" value="button_listener" />
	</node>
</launch>
